sched/signal/sig_tgkill.c

```diff
@@ -67,14 +65,13 @@
  *
  ****************************************************************************/
 
-int pthread_kill(pthread_t thread, int signo)
+int tgkill(pid_t pid, pid_t tid, int signo)
```

Comment:
Do we need to add `UNUSED(pid);` or put 
```
#ifdef CONFIG_SCHED_HAVE_PARENT
  if (pid  != (pid_t)-1)
    {
      DEBUGASSERT(pid == rtcb->pid);
    }
#endif
```
?

Reply:
Can not, since pid mean the main task pid, not the calling thread tid if it isn't -1.